src/Nethermind/Nethermind.JsonRpc/Converters/TxReceiptConverter.cs

```diff
@@ -13,7 +13,7 @@ public class TxReceiptConverter : JsonConverter<TxReceipt>
     {
         public override void WriteJson(JsonWriter writer, TxReceipt value, JsonSerializer serializer)
         {
-            serializer.Serialize(writer, new ReceiptForRpc(value.TxHash!, value, UInt256.Zero));
+            serializer.Serialize(writer, new ReceiptForRpc(value.TxHash!, value, new(UInt256.Zero)));
```

Comment:
Do we need to use `new(UInt256.Zero)` here?

Reply:
It spawns `TxGasInfo` struct that is a container for gas fields, do you feel like we can deal without it?